Rebuild-Black-Business/RBB-Website
# added algolia credits

# Describe your PR
Add Algolia credit code

Related to #
Fixes #

## Pages/Interfaces that will change

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

![image](https://user-images.githubusercontent.com/8475305/84347130-c440d180-ab7f-11ea-9b2f-04e1feb53ba0.png)


## Steps to test

1. go to /businesses
2. check bottom right of grid above pagination
3. make sure clicking on it sends you to their site

### Additional notes
